### PR TITLE
add templates choice form_submitaction_sendemail

### PR DESCRIPTION
**Pull request by qneyrat**
[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | x
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

-> issue #3226 

Add new field email.send.selectemails (in red) in ```Forms > Actions > Send form results``` for style result form mail.

Selected email must contains tokens {formfield=***}.

#### Steps to test this PR:
1.  in form config create `send form results` action.
Imgur
2. add template mail.
Imgur
3. submit form `http://yourmautic.io/form/{id}`.
4. check your email ! :)

### CODE REVIEW

File: app/bundles/FormBundle/EventListener/FormSubscriber.php [hunk 1]
```diff
@@ -197,6 +197,15 @@ public function onFormSubmitActionSendEmail(Events\SubmissionEvent $event)
             $this->mailer->setSubject($config['subject']);
 
             $this->mailer->addTokens($tokens);
+
+            if (isset($config['templates'])) {
+                /** @var \Mautic\EmailBundle\Model\EmailModel $model */
+                $model = $this->mailer->getFactory()->getModel('email');
```
Comment (dongilbert) [inline app/bundles/FormBundle/EventListener/FormSubscriber.php]: Please inject the email model directly into the FormSubscriber class using a constructor parameter. It also needs to be added to the FormSubscriber config in the bundles/FormBundle/Config/config.php section that defines the parameters for the FormSubscriber (under the `events` key).

Comment (qneyrat) [inline app/bundles/FormBundle/EventListener/FormSubscriber.php]: i fix it

**Comment by dongilbert** [comment]
Awesome @qneyrat! Thanks for making those changes

**Comment by qneyrat** [comment]
With the advice of @alanhartless, i have add js for inject html of template in `Message` input  
#

Imgur

#
So I deleted my changes in `FormSubscriber.php`

**Comment by mqueme** [comment]
This worked nicely for me. 👍

**Comment by dongilbert** [comment]
Works as described. Thanks!

**Pull request merged**